modules/authentication/Interceptors.spec.js

```diff
@@ -121,11 +121,18 @@ describe('authentication/Interceptors', function() {
 
     it('does not dispatch a sign out action on a non-401 error', function() {
       const expectedError = new Error();
-      expectedError.response = {};
+      expectedError.response = { status: 422 };
+
+      const saveTokenInfo = this.sandbox.stub(
+        Interceptors.prototype,
+        'saveTokenInfo'
+      ).callsFake(() => {});
```

Comment:
This is actually why the tests are failing on this. This is a feature of the newest version of sinon. Should be the third parameter in this version of sinon